src/Compilers/Test/Core/Compilation/CompilationExtensions.cs

```diff
@@ -30,7 +32,13 @@ namespace Microsoft.CodeAnalysis.Test.Utilities
 {
     public static class CompilationExtensions
     {
-        internal static bool EnableVerifyIOperation { get; } = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ROSLYN_TEST_IOPERATION"));
+        internal static bool EnableVerifyIOperation { get; } =
+#if ROSLYN_TEST_IOPERATION
```

Comment:
what benefit does this offer? is it just found to be easier to introduce the `#define` for local development versus adding an environment variable to the local machine or in IDE?

Reply:
also, I realize this is still in draft mode, so.. feel free to ignore this.

Reply:
> is it just found to be easier to introduce the #define for local development versus adding an environment variable to the local machine or in IDE?

Absolutely. You have to restart VS to get it to see the environment variable.